client/pages/OriginalDataPoint/components/Comments/CommentsEditor.tsx

```diff
@@ -18,6 +17,7 @@ const CommentsEditor: React.FC<Props> = (props) => {
   const textareaRef = useRef<HTMLTextAreaElement>(null)
   const descriptionEditor = useRef(null)
 
+  // TODO: CKEditor removed from project, use MarkdownEditor or MarkdownPreview component
```

Comment:
we need to use the editor in ODP Comments. separate issue ?

Reply:
Separate issue

Reply:
created #1437